pyteal/ast/router_test.py

```diff
@@ -378,6 +378,36 @@ def test_bare_call_config_clear_state_failure():
     assert "Attempt to construct clear state program from bare app call" in str(tie)
 
 
+def test_bare_call_actions_asdict():
+    no_action = pt.OnCompleteAction()
+    del_action = pt.OnCompleteAction(action=pt.Int(1), call_config=pt.CallConfig.ALL)
+    close_action = pt.OnCompleteAction(action=pt.Int(2), call_config=pt.CallConfig.CALL)
+
+    bca = pt.BareCallActions(
+        delete_application=del_action,
+        close_out=close_action,
+    )
+
+    bcad = bca.asdict()
+    assert set(bcad.keys()) == {
+        "clear_state",
+        "close_out",
+        "delete_application",
+        "no_op",
+        "opt_in",
+        "update_application",
+    }
+    with pt.TealComponent.Context.ignoreExprEquality():
```

Comment:
I believe we'd want to test without this, since the expressions should be the same